mms/model_server.py

```diff
@@ -65,6 +65,10 @@ def start():
                 print("Removing orphan pid file.")
                 os.remove(pid_file)
 
+        if (os.environ.get("OMP_NUM_THREADS") == None):
```

Comment:
Does this work? 

Also we have the same logic on [python worker](https://github.com/awslabs/multi-model-server/blob/master/mms/model_service_worker.py#L42), whats the difference?

Reply:
closing PR as per discussion